eldritch/events.py

```diff
@@ -4188,7 +4243,7 @@ def resolve(self, state):
       state.event_stack.append(self.return_monsters)
       return
 
-    if not self.can_seal:
+    if not (self.can_seal and state.get_override(self, "can_seal")):
```

Comment:
I'm not sure what I was thinking here... is it redundant or just very careful having the check both here and at event creation?